lock_diff.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "octokit", "~> 4.0"
   spec.add_dependency "httpclient"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "> 1.15"
```

Comment:
I got the following error when I tried running `bundle install` with Bundler v2.1.4.

```console
$ bundle
Fetching gem metadata from https://rubygems.org/........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler (~> 1.15)

  Current Bundler version:
    bundler (2.1.4)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

Could not find gem 'bundler (~> 1.15)' in any of the relevant sources:
  the local ruby installation
```